### ISSUE TITLE
Item Frames not culling

### ISSUE THREAD

**Issue by fyrmftw** [VolmitSoftware/React#439]
https://github.com/VolmitSoftware/React/issues/430

I'm using fresh config, latest react plugin and made sure Item frame is in the culling list, and it's still not being able to cull item frames, it works with mobs, minecarts, paintings etc, but still not the item frames.

In the image below, I placed over 30+ Paintings and itemframes and only the paintings got removed, i've tried a force cull on entities and itemframes specifically and nothing.

![image](https://user-images.githubusercontent.com/34957892/43048462-b7cead42-8ddf-11e8-9427-cb41c38b4963.png)

**Comment by cyberpwnn** [comment]
This might be hard coded. Ill add an option to disable item frame prevention.

**Comment by cyberpwnn** [comment]
Added two options (Implemented in 6.572)
```yml
features:
  react:
    saftey:

      # Treats itemframes as dangerous, and prevents react from killing them, or
      # culling them. Set this to false to allow culling them with rules, or purging
      # them.
      treat-itemframes-dangerous: true

      # Treats armorstands as dangerous, and prevents react from killing them, or
      # culling them. Set this to false to allow culling them with rules, or purging
      # them.
      treat-armorstands-dangerous: true
```

**Comment by fyrmftw** [comment]
Thank you, it culls now <3
